Changes for review ~ Christopher

FulChr3356 opened the pull request.

FulChr3356 committed:
Created Spring/React Project

FulChr3356 committed:
Added login and registration pages.

FulChr3356 committed:
Added dashboard

FulChr3356 committed:
Added database and models with dashboard

FulChr3356 committed:
Added Abstract JPA DAO

FulChr3356 committed:
Added all entities and repositories with test

FulChr3356 committed:
Added employeeController with test

FulChr3356 committed:
Add basic Entity controller methods for all entities.

FulChr3356 committed:
updated gitignore

FulChr3356 committed:
Removing all files in .gitignore

FulChr3356 committed:
Added controllers for all entities.

FulChr3356 committed:
Added find by ID test

Git-Leon reviewed (commented):
Looks like there are 30604 files in this repository.
Did you invoke `npm install` and then push up the build 👀 ??
Ensure that you fix you `.gitignore` and execute `git rm -r --cached .` first.

FulChr3356 committed:
Removing all files in .gitignore

FulChr3356 commented:
@Git-Leon Removed node_modules folder. Not sure if that'll break anything for people who clone though.

FulChr3356 committed:
Linked java to react mapping.

FulChr3356 committed:
Fixed list view

FulChr3356 committed:
Got routing to work.

FulChr3356 committed:
Added DepartmentList.js

FulChr3356 committed:
added materialUI

FulChr3356 committed:
fixed more stuff. added budget edit

FulChr3356 committed:
fixed css on dashboard

FulChr3356 committed:
added authentication via okta oauth2

FulChr3356 committed:
Login working w/o password, added react-cookie and redirect cookie

FulChr3356 committed:
Login working w/o password, added react-cookie and redirect cookie

FulChr3356 committed:
edited routing and crud mapping. Added Payroll page with controller.

FulChr3356 committed:
added pie chart, added @data lombok,everaged  auto create tables with EL

FulChr3356 committed:
added pie chart, added data lombok,everaged  auto create tables with EL

FulChr3356 committed:
Merge remote-tracking branch 'origin/master'

FulChr3356 commented:
@Git-Leon Added quite a bit, general comments or critic welcomed.

FulChr3356 committed:
fixed some stuff broke more 

changed to table for empoyee and department, TODO the rest.redirect for login and logout

FulChr3356 committed:
Fixed Join table reference with employee_department  add Item model

Removed budget model and instead made it a model variable for each department

FulChr3356 committed:
v0.1 changed auth method, all crud operations operational

FulChr3356 committed:
Deleted unnecessary files

FulChr3356 committed:
Added heroku mariadb database info

FulChr3356 committed:
disabled cors

FulChr3356 committed:
changed url

FulChr3356 committed:
changed url again

FulChr3356 commented:
Automatic deployment with Heroku implemented URL: [https://upkeep-assistant.herokuapp.com](https://upkeep-assistant.herokuapp.com)
TODO: bug fixes

FulChr3356 committed:
Fixed some UI bugs

FulChr3356 committed:
Update application.properties

FulChr3356 committed:
add static.json and heroku build

FulChr3356 committed:
Merge remote-tracking branch 'origin/master'

FulChr3356 committed:
Auto stash before merge of "master" and "origin/master"

FulChr3356 committed:
Update README.md

FulChr3356 committed:
added alerts

FulChr3356 committed:
Merge remote-tracking branch 'origin/master'

FulChr3356 committed:
Update auth.service.js

FulChr3356 committed:
Update user.service.js

FulChr3356 committed:
Push for my boi Leon

FulChr3356 committed:
Split components with react-router, fixed employee_department bug.

FulChr3356 committed:
Deleting yarn.lock

FulChr3356 committed:
Update auth.service.js

FulChr3356 committed:
Update auth.service.js

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update auth.service.js

FulChr3356 committed:
Update auth.service.js

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update auth.service.js

FulChr3356 committed:
Update auth.service.js

FulChr3356 committed:
Update package.json

FulChr3356 committed:
Update static.json

FulChr3356 committed:
Delete static.json

FulChr3356 committed:
Update package.json

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update EmployeeList.js

FulChr3356 committed:
Update application.properties

FulChr3356 committed:
Update README.md

FulChr3356 committed:
Update README.md

FulChr3356 committed:
Update README.md

Git-Leon committed:
Update README.md

Git-Leon committed:
Update README.md

Git-Leon committed:
Update README.md
